webpack.config.client.js

```diff
@@ -16,6 +16,19 @@ module.exports = ({ resolvePath, IS_PROD, START_DEV_SERVER }) => {
       port: webpackDevServerPort,
       historyApiFallback: true,
       hot: true,
+      stats: {
+        assets: false,
+        children: false,
+        chunks: false,
+        hash: false,
+        modules: false,
+        publicPath: false,
+        timings: false,
+        version: false,
+        warnings: true,
+        colors: true,
```

Comment:
I see why we don't need to pass `--colors` to the `webpack:dev:client` script ^

Could you add:

```
stats: {
  colors: true, // color the console output in terminal
},
```

... to the base webpack.config.js and remove it here? (Should also mean you don't need to pass `--colors` on the CLI anymore).

Reply:
In fact we could move this whole `stats` block up to webpack.config.js so we don't have to pass `--hide-modules` either.